src/Aspire.Dashboard/Components/Pages/Token.razor

```diff
@@ -0,0 +1,40 @@
+﻿@page "/token"
+@layout EmptyLayout
+@attribute [AllowAnonymous]
+
+<PageTitle>.NET Aspire Token Validation</PageTitle>
```

Comment:
All other pages are "{ApplicationName} resources", "{ApplicationName} structured logs", etc.

IMO make this "{ApplicationName} log in"